### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by Bovive** [axewater/sharewarez#42] [Q&A]
Where is the Docker image hosted? Not super familiar with how that works. I was curious if it had its own Docker page.

**Comment by axewater** [comment]
yes there is 'docker hub', and I did upload some versions there. but I need to rebuild the docker and upload it there every time, which I have not done in ages. I don't see an advantage to using docker hub since a docker compose will just grab the latest code from the git page anyway.

**Reply by Bovive** [reply]
Docker compose only grabs the image. The Docker compose file here grabs 1.2.1, but that's also the latest release. So perhaps that is why.
